tests/PySys/log_request_end_to_end/log_request_error_messages/run.py

```diff
@@ -43,20 +43,22 @@ def validate(self):
 
     @retry(Exception, tries=20, delay=1)
     def wait_until_logs_retrieved(self):
-       
+
         log_file = self.cumulocity.retrieve_log_file(self.operation_id)
         if len(log_file) != 0:
             return self.download_file_and_verify_error_messages(log_file)
         else:
             raise Exception("retry")
-      
+
     def create_logs_for_test(self):
-        log = self.startProcess(
-            command=self.sudo,
-            arguments=[
-                "python3", f"{os.getcwd()}/log_request_end_to_end/create_test_logs.py"],
-            stdouterr="log_failed",
-        )
+        # remove if there are any old files
+        os.system("sudo rm -rf /tmp/sw_logs")
```

Comment:
Using `self.startProcess()` is recommended over `os.system()` calls as it will give us access to the stdout and stderr outputs in the case of a failure. But here, I understand that the possibility of a failure is rare. I also vaguely remember you saying that `startProcess` was not working in some cases. Was that the case here as well? If that's the case, I'm really curious why a simple file removal wouldn't work with `startProcess`. Hope you were using `sudo` in the `startProcess()` calls as well.

Reply:
Fixed